crates/core/liveview-native-core-swift/Sources/LiveViewNativeCore/Support.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+
+class SimpleHandler: DocumentChangeHandler {
+    var callback: (Document, NodeRef) -> ()
```

Comment:
Doesn't look like this is ever mutated, unless the `DocumentChangeHandler` requires it (is that a base class or a protocol?)
```suggestion
    let callback: (Document, NodeRef) -> ()
```